<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-OGUBGQ1C" ignored="0" name="Test 2" tests="2" failures="1" timestamp="2022-04-19T18:26:56 IST" time="9.195" errors="0">
  <testcase name="login" time="1.159" classname="com.sevenrmart.qa.testcases.VerifyingLoginPageTest"/>
  <testcase name="loginverification" time="0.102" classname="com.sevenrmart.qa.testcases.VerifyingLoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [https://groceryapp.uniqassosiates.com/admin/login] but found [https://groceryapp.uniqassosiates.com/admin]">
      <![CDATA[java.lang.AssertionError: expected [https://groceryapp.uniqassosiates.com/admin/login] but found [https://groceryapp.uniqassosiates.com/admin]
at com.sevenrmart.qa.testcases.VerifyingLoginPageTest.loginverification(VerifyingLoginPageTest.java:26)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- loginverification -->
</testsuite> <!-- Test 2 -->
